cmd/fluent-watcher/fluentd/Dockerfile.arm64

```diff
@@ -0,0 +1,87 @@
+# Fluentd watcher agent
+FROM golang:1.16.6-alpine3.13 as buildergo
+RUN mkdir -p /fluentd
+RUN mkdir -p /code
+COPY . /code/
+WORKDIR /code
+RUN echo $(ls -al /code)
+RUN CGO_ENABLED=0 go build -i -ldflags '-w -s' -o /fluentd/fluentd-watcher /code/cmd/fluent-watcher/fluentd/main.go
+
+# To set multiarch build for Docker hub automated build.
+FROM golang:alpine AS builderqemu
+WORKDIR /go
+ENV QEMU_DOWNLOAD_SHA256 5db25cccb40ac7b1ca857653b883376b931d91b06ff34ffe70dcf6180bd07bb8
+RUN apk add curl --no-cache
+RUN curl -sL -o qemu-6.0.0.balena1-aarch64.tar.gz https://github.com/balena-io/qemu/releases/download/v6.0.0%2Bbalena1/qemu-6.0.0.balena1-aarch64.tar.gz && echo "$QEMU_DOWNLOAD_SHA256 *qemu-6.0.0.balena1-aarch64.tar.gz" | sha256sum -c - | tar zxvf qemu-6.0.0.balena1-aarch64.tar.gz -C . && mv qemu-6.0.0+balena1-aarch64/qemu-aarch64-static .
+
+# Fluentd main image
+FROM arm64v8/ruby:2.7-slim-bullseye
```

Comment:
Can we just `From fluent/fluentd:v1.14.5-debian-arm64-1.0` and then add fluent-watcher files and then use a new entrypoint instead of copying all the Dockerfile for arm64 fluentd?

Reply:
I have tested `gem install` steps that take more than 10 mins.  And we have extra pkg to install.  I think it would help, but it will not improve build time. Let the user build it themselves.